_episodes/06-raster-intro.md

```diff
@@ -0,0 +1,433 @@
+---
```

Comment:
Indeed. I have documented this in #21. Maybe we can finalize the content first and update the table of contents in one go later.

Reply:
sounds good to do this in a separate pr. there might be a conflict with the episode @fnattino just worked on since that is also listed as episode 5 like this one is right now. I'm not sure how TOC order is assigned but we can fix it once all the content prs are in